include/geode/mesh/core/grid.hpp

```diff
@@ -175,7 +175,7 @@ namespace geode
                 { 2, 6 }, { 3, 7 } } };
 
         static constexpr std::array< std::array< geode::local_index_t, 4 >, 6 >
-            CELL_FACET_VERTICES_3D{ { { 0, 4, 6, 2 }, { 1, 5, 7, 3 },
+            CELL_FACET_VERTICES_3D{ { { 0, 2, 6, 4 }, { 1, 5, 7, 3 },

```

Comment:
sure about this change?

Reply:
Yes, this was a bug: the facet was not oriented like the others